workers/social/lib/social/models/group/index.coffee

```diff
@@ -495,18 +475,8 @@ module.exports = class JGroup extends Module
 
   @create$ = secure (client, formData, callback) ->
     { delegate } = client.connection
-
-    # subOptions = targetOptions: selector: tags: "custom-plan"
-    # delegate.fetchSubscription null, subOptions, (err, subscription) =>
-    #   return callback err  if err
-    #   return callback new KodingError "Subscription is not found"  unless subscription
-    #   subscription.debitPack tag: "group", (err) =>
-    #     return callback err  if err
     @create client, formData, delegate, (err, group) ->
-      return callback err if err
-      # group.addSubscription subscription, (err) ->
-      #   return callback err  if err
-      callback null, { group }
+      return callback err, group
```

Comment:
just this one questioned me, checked it out site-wide it seems it's never used like `group.group` not sure how it was working :)